src/components/InputandFilterPanel/InputandFilterPanel.styles.tsx

```diff
@@ -67,16 +67,57 @@ export const InputImage = styled.img`
 	max-height: 15px;
 `;
 
-export const StyledCountriesSelectLabel = styled.label`
+export const StyledCountriesLabel = styled.label`
 	border-radius: 5px;
 	width: 20%;
 	min-width: 145px;
 	background-color: ${(props) => props.theme.colors.white};
 	text-align: center;
 `;
 
-export const StyledCountriesSelect = styled.select`
+export const StyledFilterPanel = styled.div`
+	border-radius: 5px;
+	background-color: ${(props) => props.theme.colors.white};
+	width: 15%;
+	min-width: 170px;
+	display: flex;
+	justify-content: space-around;
+	padding: 20px 10px;
+	box-shadow: 2px 2px 10px -8px rgba(66, 68, 90, 1);
+	font-family: ${(props) => props.theme.fonts.family.basic};
+	font-weight: ${(props) => props.theme.fonts.weight.semiBold};
+	font-size: ${(props) => props.theme.fonts.sizes.s};
+	position: relative;
+`;
+
+export const FilterButton = styled.button`
 	border: none;
-	width: 90%;
-	padding: 18px 5px;
+	outline: none;
```

Comment:
Why did you remove the outline?

Reply:
removed it. I used it to remove outline border around button element.